.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        version: [ 20.10.5-buster-slim, 21.10.0-buster-slim, ci ]
+        version: [ previous-lts, lts, latest ]
```

Comment:
why `latest` and not `ci`?

Reply:
@YoEight can the `latest` be changed to `ci` again?

Reply:
Done